### PR TITLE
po: Added german translation

### PR DESCRIPTION
**Pull request by stenosis**
Just created a PO file

### CODE REVIEW

**Review by tchx84 — APPROVED** [review]
Thank you, ill merge it after a quick test, I just ran out of time now.

**Comment by stenosis** [comment]
My pleasure. There is no hurry :-)

I had found that the close button on the about dialog didn't worked for me. So I just looked at GTK3's JS bindings for the first time to find this bug. This commit was accidentally added. I hope you dont mind, sorry.

**Comment by tchx84** [comment]
@stenosis , Thanks. Please move that fix to a separate PR ? So we can discuss that issue separately.

**Comment by stenosis** [comment]
Sure, sounds good :-)

File: po/de.po [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+msgid "Access bluetooth"
+msgstr "Zugriff auf Bluetooth"
+
+#: src/model.js:110
+msgid "Access other syscalls (e.g. ptrace)"
+msgstr "Zugriff auf andere Systemaufrufe (z.B. ptrace)"
+
+#: src/model.js:115
+msgid "Access programs from other architectures"
+msgstr "Zuriff auf Programme anderer Architekturen"
```
Comment (tchx84) [inline po/de.po]: I noticed while testing `Zuriff`, is this a typo?

**Review by tchx84 — CHANGES_REQUESTED** [review]

File: po/de.po (hunk 1 above)
Comment (stenosis) [reply po/de.po]: Oh, you're right. It's a typo. It should be called "Zugriff" with a 'g'.

**Comment by tchx84** [comment]
Thanks!

**Pull request merged**